vendor/engines/split_accounts/app/services/split_accounts/reports/querier.rb

```diff
@@ -0,0 +1,29 @@
+module SplitAccounts
+
+  module Reports
+
+    class Querier < ::Reports::Querier
+
+      def perform
+        super.map do |order_detail|
+          simulate_order_details(order_detail)
+        end.flatten
+      end
+
+      def simulate_order_details(order_detail)
+        if order_detail.account.is_a?(SplitAccounts::SplitAccount)
```

Comment:
I originally added `splits` to the `Account` model.  But then decided against it since that means the account split gem is bleeding into the main app.  I like your idea to move it into the `SplitOrderDetailSimulator` object (or whatever we end up calling it.)